packages/lib/src/components/Dropin/Dropin.tsx

```diff
@@ -12,10 +12,11 @@ import { hasOwnProperty } from '../../utils/hasOwnProperty';
 import SRPanelProvider from '../../core/Errors/SRPanelProvider';
 import splitPaymentMethods from './elements/splitPaymentMethods';
 import type { ICore } from '../../core/types';
+import { IDropin } from './types';
```

Comment:
Not a big deal, but I'd recommend to start using `import type { isDropin } from './types';` instead of direct import when importing typescript types. 

It helps making sure we are always importing only the 'type' and that the imported values will be eliminated during compilation. ([more here](https://medium.com/@quizzesforyou/import-vs-import-type-in-typescript-8e5177b62bea))

Reply:
Do you mean `IDropin` rather than `isDropin`? :-P